content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -377,6 +377,7 @@ different Kubernetes components.
 | `IngressClassNamespacedParams` | `true` | GA | 1.23 | - |
 | `Initializers` | `false` | Alpha | 1.7 | 1.13 |
 | `Initializers` | - | Deprecated | 1.14 | - |
+| `JobPodFailurePolicy` | `false` | Alpha | 1.25 | - |
```

Comment:
Remember to define this feature gate at the end of the file.

Reply:
Updated